code/__DEFINES/melee.dm

```diff
@@ -0,0 +1,10 @@
+//Martial arts defines
+
+#define MARTIALART_BOXING "boxing"
+#define MARTIALART_WRESTLING "wrestling"
+#define MARTIALART_SLEEPINGCARP "sleeping carp"
+#define MARTIALART_PSYCHOBRAWL "psychotic brawling"
```

Comment:
These are display names and shouldn't be lowercased

Reply:
Changes made, they're now used in an ID variable and never seen by the mortal eye.